### DISCUSSION TITLE
WebAssembly - From Browser to Cloud

### DISCUSSION THREAD

**Discussion by giscus** [PradeepLoganathan/pradeepl-blog#11] [BlogPosts]
# WebAssembly - From Browser to Cloud 

WebAssembly(WASM) is a low-level assembly-like language that is designed for efficient execution and compact representation. WebAssembly is a binary-encoded instruction format or bytecode that runs on a stack-based virtual machine. WebAssembly systems interface(WASI) is a runtime specification and a standard for WebAssembly interaction with the operating system.

https://pradeepl.com/blog/webassembly-from-browser-to-cloud/

**Comment by ngurusamy** [comment]
Hi Pradeep,

I am Guru. you blog was information to me as I have started learning webassembly.

Currently, We are in the plan of porting activeX framework being used for  browser vpn plugin into webassembly. The current plugin uses the required method(ex. encrypt, decrypt..) defined in the DLL file. I am not certain that all could be done in webassembly... 

Need your suggestion/guidance in this...

Thanks in Advance.

Guru.
